BDArmory/FX/ExplosionFX.cs

```diff
@@ -472,8 +468,13 @@ private bool IsInLineOfSight(Part part, Part explosivePart, out RaycastHit hit,
                         }
                         if (FlightGlobals.currentMainBody != null && hit.collider.gameObject == FlightGlobals.currentMainBody.gameObject) return false; // Terrain hit. Full absorption. Should avoid NREs in the following.
                         var partHP = partHit.Damage();
-						if (ProjectileUtils.IsArmorPart(partHit)) partHP = 100;
-                        var partArmour = partHit.GetArmorThickness();
+			if (ProjectileUtils.IsArmorPart(partHit)) partHP = 100;
```

Comment:
I suspect this line is the reason you were seeing armor panels return subpar damage reduction; they don't get the (potentially very large) HP bulk of standard parts to resist blast